projects/akita-ng-fire/src/lib/collection/collection.service.ts

```diff
@@ -213,7 +213,7 @@ export class CollectionService<S extends EntityState<EntityType, string>, Entity
     }
     // Start Listening
     return this.db.collection<EntityType>(path, queryFn).stateChanges().pipe(
-      withTransaction(actions => syncStoreFromDocAction(storeName, actions, this.idKey, (entity) => this.formatFromFirestore(entity)))
+      withTransaction(actions => syncStoreFromDocAction(storeName, actions, (entity) => this.formatFromFirestore(entity)))
```

Comment:
Why you remove the idKey ? Let's say your user has a idkey "uid" instead of "id" will it create an issue ?